packages/core/src/xr/IXRDevice.ts

```diff
@@ -0,0 +1,35 @@
+import { IXRPlatformFeature } from "@galacean/engine-design";
+import { Engine } from "../Engine";
+import { XRSessionType } from "./session/XRSessionType";
+import { XRInputManager } from "./input/XRInputManager";
+import { XRSessionManager } from "./session/XRSessionManager";
+
+/**
+ * The interface of XR device.
+ */
+export interface IXRDevice {
```

Comment:
Backend interface should in design package

Reply:
Done.